### PR TITLE
Add the "google_compute_global_address" datasource

### PR DESCRIPTION
**Pull request by eilgin**
Same as #748. This PR will be useful for users that want to expose the public IP of their HTTP load balancers.

### CODE REVIEW

**Comment by eilgin** [comment]
Acceptance result:
```bash
TF_ACC=1 go test $(go list ./... | grep -v 'vendor') -v -run=TestAccDataSourceComputeGlobalAddress -timeout 120m
?       github.com/terraform-providers/terraform-provider-google        [no test files]
=== RUN   TestAccDataSourceComputeGlobalAddress
--- PASS: TestAccDataSourceComputeGlobalAddress (26.33s)
PASS
ok      github.com/terraform-providers/terraform-provider-google/google 26.362s
testing: warning: no tests to run
PASS
ok      github.com/terraform-providers/terraform-provider-google/scripts        0.028s [no tests to run]
```

File: google/data_source_google_compute_global_address_test.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+		if ds_attr["status"] != "RESERVED" {
+			return fmt.Errorf("status is %s; want RESERVED", ds_attr["status"])
+		}
+
+		return nil
+	}
+}
+
+func testAccCheckDataSourceComputeGlobalAddressDestroy(resource_name string) resource.TestCheckFunc {
```
Comment (rosbo) [inline google/data_source_google_compute_global_address_test.go]: You should be able to get the same result by using this method instead: `testAccCheckComputeGlobalAddressDestroy`

Comment (eilgin) [reply google/data_source_google_compute_global_address_test.go]: i've "fixup" the code

**Comment by rosbo** [comment]
```sh
--- PASS: TestAccDnsRecordSet_basic (22.47s)
```

**Pull request merged**